docs/src/main/sphinx/installation/jdbc.rst

```diff
@@ -120,92 +120,97 @@ may not be specified using both methods.
-                                                             ``catalog1:roleA;catalog2:roleB`` sets ``roleA``
-                                                             for ``catalog1`` and ``roleB`` for ``catalog2``.
-``sessionProperties``                                        Session properties to set for the system and for catalogs,
-                                                             specified as a list of key-value pairs.
-                                                             For example, ``abc:xyz;example.foo:bar`` sets the system property
-                                                             ``abc`` to the value ``xyz`` and the ``foo`` property for
-                                                             catalog ``example`` to the value ``bar``.
-``externalAuthentication``                                   Set to true if you want to use external authentication via
-                                                             :doc:`/security/oauth2`. Use a local web browser to authenticate with an
-                                                             identity provider (IdP) that has been configured for the Trino coordinator.
-``externalAuthenticationTokenCache``                         Allows the sharing of external authentication tokens between different
-                                                             connections for the same authenticated user until the cache is
-                                                             invalidated, such as when a client is restarted or when the classloader
-                                                             reloads the JDBC driver. This is disabled by default, with a value of
-                                                             ``NONE``. To enable, set the value to ``MEMORY``. If the JDBC driver is used
-                                                             in a shared mode by different users, the first registered token is stored
-                                                             and authenticates all users.
-``disableCompression``                                       Whether compression should be enabled.
-``assumeLiteralNamesInMetadataCallsForNonConformingClients`` When enabled, the name patterns passed to ``DatabaseMetaData`` methods
-                                                             are treated as literals. You can use this as a workaround for
-                                                             applications that do not escape schema or table names when passing them
-                                                             to ``DatabaseMetaData`` methods as schema or table name patterns.
-============================================================ =======================================================================
+================================================================= =======================================================================
```

Comment:
Split changes into two commits

1. that adds more whitespace to this table (and no other changes)
2. second adds the new property and deprecates the old one